src/linear_algebra/alternating.lean

```diff
@@ -870,3 +894,58 @@ begin
 end
 
 end basis
+
+/-! ### Currying -/
+
+section currying
+
+variables
+  {R' : Type*} {M'' N'' : Type*}
+  [comm_semiring R'] [add_comm_monoid M''] [add_comm_monoid N'']
+  [module R' M''] [module R' N'']
+
+namespace alternating_map
+
+/-- Given an alternating map `f` in `n+1` variables, split the first variable to obtain
+a linear map into alternating maps in `n` variables, given by `x ↦ (m ↦ f (matrix.vec_cons x m))`.
+
+This is `multilinear_map.curry_left` for `alternating_map`. -/
```

Comment:
```suggestion
This is `multilinear_map.curry_left` for `alternating_map`. It can be thought of as a map
$Hom(\wedge^{n+1} M, N) \to Hom(M, Hom(\wedge^n M, N))$. See also `alternating_map.curry_left_linear_map`. -/
```

Reply:
Went for `bigwedge` instead, but otherwise applied.